tests/simulate_telemetry.py

```diff
@@ -55,9 +55,11 @@
 }
 
 parameters = [
```

Comment:
Add tests for all fingerprints as some are not tested yet

Reply:
@Scienfitz do you think we need to adapt this or would be what I added to `test_fingerprints.py` enough - just testing that all can be run from smiles to embedding?

Reply:
a bit confused about the commen being on this fiel `simulate_telemetry`

as for fingerprint parameterization: once you wrote a test you can simply parameterize it via pytest. For instance I can imagine a tets looping over i) all fingperrints available, ie parameterized via `SubstanceEncoding` so it automatically always runs over all valdi choices - with default kwargs etc PLUS ii) some selected fps like ECFP run with special kwargs to also test different nbits and radii

Reply:
i) SubstanceEncoding values are samples in `hypothesis_strategies/parameters.py`
ii) Added a few tests with different kwargs in `tests/test_fingeprints.pt`